test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -447,6 +447,23 @@ def forward(self, input):
                       input_names=['input_1'],
                       dynamic_axes={'input_1': [0, 1, 2]})
 
+    def test_tensor(self):
```

Comment:
Can you add a test case where tensor list contains a combination of tensors of scalars and 1D input?

like:
```
        class TensorInputModel(torch.jit.ScriptModule):
            @torch.jit.script_method
            def forward(self, input):
                return torch.tensor([input.shape[0], int(input)])

        x = torch.randn(1)
        self.run_test(TensorInputModel(), x)
```

Reply:
Added this test

Reply:
Thanks.
Maybe also add a test where dtype is provided:

```
        class TensorInputModel(torch.jit.ScriptModule):
            @torch.jit.script_method
            def forward(self, input):
                return torch.tensor([input.shape[0], input.shape[1]], dtype=torch.float)
```